packages/shared/lib/core/profile/constants/default-persisted-profile-object.constant.ts

```diff
@@ -21,6 +22,6 @@ export const DEFAULT_PERSISTED_PROFILE_OBJECT: IPersistedProfile = {
     accountPersistedData: {},
     isDeveloperProfile: false,
     forceAssetRefresh: false,
-    strongholdVersion: undefined,
+    strongholdVersion: StrongholdVersion.V2,
```

Comment:
We should not set our default to v2 in my opinion, shoul dbe v3 because all new profiles will be v3

Reply:
We should assume the user comes with an V2 as with don't know the exact version

- User profile comes with V2 -> We try to migrate, no error is thrown, profile is migrated and we set the profile to v3
- User profile comes with V3 -> We try to migrate, and error is thrown internally but it's skipped, we simply set the profile to v3

Reply:
`DEFAULT_PERSISTED_PROFILE_OBJECT` is used during onboarding too, it doesnt make sense doing it this way, we can simply set the version to v3 in the chrysalis to stardust migration instead of affecting the entire app

Reply:
Gotcha, just updated this in https://github.com/iotaledger/firefly/pull/7555/commits/3c3bf612047c1ff524036697813416e005604217, looks good?